.drone.yml

```diff
@@ -11,12 +11,12 @@ concurrency:
 
 steps:
 - name: sync
-  image: xoxys/git-batch:latest
+  image: thegeeklab/git-batch:latest
   commands:
-  - git-batch
+  - git-batch -vvv
 
 - name: build
-  image: owncloudci/hugo:0.69.2
+  image: owncloudci/hugo:0.71.0
```

Comment:
Current upstream release is v0.75, maybe upgrade if you touch it anyway

Reply:
please not now :-) I am not sure about the implications. Need to test first.